ampersand-chart.js

```diff
@@ -545,6 +563,24 @@
           .orient('left');
 
         yAxis.call(yAxisGenerator);
+
+        (function(yAxis, chart) {
+          setTimeout(function() {
+            if (yAxis) {
+              var yAxisOffset = 0;
+              yAxis.selectAll('text').each(function() {
+                yAxisOffset = Math.max(this.getBBox().width, yAxisOffset);
+              });
+              yAxis.attr('x', yAxisOffset + 12);
+              yAxis.selectAll('line').each(function(d) {
+                if (d === 0) {
+                  d3.select(this).remove();
+                }
+              });
+              chart.select('line.ampersand-graph-ground').attr('x1', yAxisOffset + 12);
+            }
+          }.bind(this), 1);
+        }.bind(this))(yAxis, chart);
```

Comment:
Is it possible to reduce the duplicate code in here?  This block of code looks like it appears 3 times.